<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,3,1,0">
    <item row="3" column="0" colspan="2">
     <widget class="QTreeView" name="treeView"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="save">
      <property name="text">
       <string>export npy</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="load">
      <property name="text">
       <string>load fbx</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QVBoxLayout" name="modelLayout"/>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSlider" name="verticalSlider1">
        <property name="maximum">
         <number>2999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="verticalSlider2">
        <property name="maximum">
         <number>2999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="verticalSlider3">
        <property name="maximum">
         <number>2999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="verticalSlider4">
        <property name="maximum">
         <number>2999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="verticalSlider5">
        <property name="maximum">
         <number>2999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="verticalSlider6">
        <property name="maximum">
         <number>2999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="verticalSlider7">
        <property name="maximum">
         <number>2999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="verticalSlider8">
        <property name="maximum">
         <number>2999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="verticalSlider9">
        <property name="maximum">
         <number>2999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="verticalSlider10">
        <property name="maximum">
         <number>2999</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="setting"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>显示</string>
    </property>
    <addaction name="actionPOINTS"/>
    <addaction name="actionFACE"/>
    <addaction name="actionCONTOUR_LINE"/>
    <addaction name="actionFACE_CONTOUR_LINE"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="setting">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/iconImage/img/settings.png</normaloff>:/iconImage/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionPOINTS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>POINTS</string>
   </property>
  </action>
  <action name="actionFACE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FACE</string>
   </property>
  </action>
  <action name="actionCONTOUR_LINE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CONTOUR_LINE</string>
   </property>
  </action>
  <action name="actionFACE_CONTOUR_LINE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FACE_CONTOUR_LINE</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
